charts/kof-storage/templates/victoria/vmcluster.yaml

```
@@ -12,6 +13,9 @@ spec:
   vminsert:
     extraArgs: {}
     image:
+      {{- with $global.registry }}
+      repository: {{ $global.registry }}/victoriametrics/vminsert
+      {{- end }}
```

Here and in other places:
```suggestion
      {{- with $global.registry }}
      repository: {{ . }}/victoriametrics/vminsert
      {{- end }}
```